.travis.yml

```diff
@@ -9,14 +9,27 @@ go:
  - 1.5.3
  - 1.6rc2
 
+os:
+ - linux
+ - osx
+
 env:
   matrix:
   - BLAS_LIB=OpenBLAS
   - BLAS_LIB=gonum
-  # at some point, when travis allows builds on darwin
-  #- BLAS_LIB=Accellerate
-  # at some point, when the issue with drotgm is resolved
-  #- BLAS_LIB=ATLAS
+  - BLAS_LIB=Accellerate
+  - BLAS_LIB=ATLAS
+
+matrix:
+ exclude:
+  - os: linux
+    env: BLAS_LIB=Accellerate
+  - os: linux
+    env: BLAS_LIB=ATLAS
+  - os: osx
+    env: BLAS_LIB=OpenBLAS
```

Comment:
Haven't gotten the build for it to work yet.  Homebrew puts it someplace other than apt-get, and my macbook is currently not available for me to fiddle around with.  It is much easier to run locally instead of on travis with a 2-10 minute turnaround.